telemetry_api/lib/telemetry_api/operators.ex

```diff
@@ -0,0 +1,168 @@
+defmodule TelemetryApi.Operators do
+  @moduledoc """
+  The Operators context.
+  """
+
+  import Ecto.Query, warn: false
+  alias TelemetryApi.Repo
+
+  alias TelemetryApi.Operators.Operator
+  alias TelemetryApi.ContractManagers.OperatorStateRetriever
+  alias TelemetryApi.ContractManagers.DelegationManager
+
+  @doc """
+  Returns the list of operators.
+
+  ## Examples
+
+      iex> list_operators()
+      [%Operator{}, ...]
+
+  """
+  def list_operators do
+    Repo.all(Operator)
+  end
+
+  @doc """
+  Gets a single operator.
+
+  ## Examples
+
+      iex> get_operator("some_address"})
+      %Operator{}
+
+      iex> get_operator("non_existent_address")
+      nil
+
+  """
+  def get_operator(address) do
+    Repo.get(Operator, address)
+  end
+
+  @doc """
+  Fetches all operators.
```

Comment:
It would be useful to know in the docs that operator information is fetched from Ethereum, then metadata is added to them and inserted in the operator database

Reply:
Good idea! Done!